src/main/java/uk/gov/companieshouse/filinghistory/api/model/mongo/FilingHistoryDocument.java

```diff
@@ -35,12 +34,12 @@ public class FilingHistoryDocument {
     @Field("delta_at")
     @JsonProperty("delta_at")
     private String deltaAt;
-    @Field("updated_at")
-    @JsonProperty("updated_at")
-    private Instant updatedAt;
-    @Field("updated_by")
-    @JsonProperty("updated_by")
-    private String updatedBy;
+    @Field("updated")
```

Comment:
```@Field``` and ```@JsonProperty``` are only required when the intended name is different from the variable name, as they are the same the annotations can be removed, similar to "data" line 27 above.

Reply:
ah okay cool I will remove now.